components/AdminTable/AdminReusesTable/AdminReusesTable.vue

```diff
@@ -120,12 +120,14 @@ import AdminContentWithTooltip from '../../../components/AdminContentWithTooltip
 import type { Activity } from '~/types/activity'
 import type { ReuseSortedBy, SortDirection } from '~/types/types'
 
-const props = defineProps<{
+const props = withDefaults(defineProps<{
   activities: Record<string, Activity>
   reuses: Array<Reuse>
   sortedBy: ReuseSortedBy
   sortDirection: SortDirection
-}>()
+}>(), {
+  activities: () => ({}),
+})
```

Comment:
I had some issue with missing activities in topics/[id]/reuses.vue. Fixed it in https://github.com/datagouv/cdata/pull/455/commits/51bc264ace3a887aa28b0fd63dbdd6edc0ed4207